compile/core.rb

```diff
@@ -225,6 +225,16 @@ def quote_string(value)
       end
     end
 
+    def quote_value(value)
```

Comment:
This is specific to go though, since ruby uses single quotes. I think the original idea of having it in its own file was so that language-specific utilities could all be grouped together.